app/controllers/test_passages_controller.rb

```diff
@@ -14,8 +14,29 @@ def update
     end
   end
 
+  def gist
+    work = GistQuestionService.new(@test_passage.current_question).call
```

Comment:
```suggestion
    result = GistQuestionService.new(@test_passage.current_question).call
```
или `responce` ?
хотя по сути текущего кода даже так:
```suggestion
    service = GistQuestionService.new(@test_passage.current_question).call
```